### PR TITLE
Fix case when we have @id and extra one annotation in serializable model 

### PR DESCRIPTION
**Pull request by Mazel-Tovr**

### CODE REVIEW

File: src/test/kotlin/XodusTest.kt [hunk 1]
```diff
@@ -71,6 +71,21 @@ class XodusTest {
...
+        val all = agentStore.getAll<ObjectWithTwoAnnotation>()
+        assertEquals(1, all.count())
+        val foundById = agentStore.findById<ObjectWithTwoAnnotation>(id)
+        assertEquals(data, foundById)
+        val data2 = ObjectWithTwoAnnotation(id, 2000)
+        agentStore.store(data2)
+        assertEquals(1, agentStore.getAll<ObjectWithTwoAnnotation>().count())
+        @Suppress("RemoveExplicitTypeArguments")
+        assertEquals(data2, agentStore.findById<ObjectWithTwoAnnotation>(id))
+    }
```
Comment (LikhanovMaxim) [inline src/test/kotlin/XodusTest.kt:88]: what's the point to do the same things for data2?

Comment (Mazel-Tovr) [reply src/test/kotlin/XodusTest.kt:88]: we check that the second annotation does not interfere with the @id annotation, for this we first save the object in the database, find it by id and data2 is update entity update, in fact, we still need to add a case for deletion

Comment (LikhanovMaxim) [reply src/test/kotlin/XodusTest.kt:88]: ok, I got it, extract common code into a new method to remove duplicates

File: src/test/kotlin/XodusTest.kt [hunk 2]
```diff
@@ -286,4 +291,12 @@ class XodusTest {
         assertNotNull(agentStore.findById<MapField>("id3") != null)
     }
 
+    private suspend inline fun <reified T : Any> updateObject(data: T, id: Any, objectCount: Int) {
+        agentStore.store(data)
+        val all = agentStore.getAll<T>()
+        assertEquals(objectCount, all.count())
+        val foundById = agentStore.findById<T>(id)
+        assertEquals(data, foundById)
+    }
```
Comment (LikhanovMaxim) [inline src/test/kotlin/XodusTest.kt:300]:
```suggestion
    private suspend inline fun <reified T : Any> updateObject(objectValue:Int = 100, id: Any, objectCount: Int = 1) {
        val data = ObjectWithTwoAnnotation(id, objectMeta)
        agentStore.store(data)
        val all = agentStore.getAll<T>()
        assertEquals(objectCount, all.count())
        val foundById = agentStore.findById<T>(id)
        assertEquals(data, foundById)
    }
```

**Review by LikhanovMaxim — APPROVED** [review]

**Pull request merged**